br-new

```diff
@@ -0,0 +1,230 @@
+br:
+  profile_help: |
```

Comment:
To match up with the other keys in files such as `help.en.yml`, this should be changed

```suggestion
  profile: |
```